metadata_tools/hosts/GO_0xxx/GO_0xxx_geometry_cloud.py

```
@@ -0,0 +1,74 @@
+#
+# For local runs, the basic usage is identical to GO_0xxx_geometry.py. In addition, all
+# cloud_tasks arguments are accepted. For example:
+#
+#   python GO_0xxx_geometry_cloud.py $RMS_METADATA_TEST/GO_0xxx/ $RMS_METADATA/GO_0xxx/ $RMS_METADATA_TEST/GO_0xxx/ \
+#                                       --num-simultaneous-tasks 12 --max-runtime 10000
+#
+#
+#
+#########################################################################################
+import asyncio
+import os, sys
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)